interface/kaHIP_interface.h

```diff
@@ -8,12 +8,21 @@
 #ifndef KAFFPA_INTERFACE_RYEEZ6WJ
 #define KAFFPA_INTERFACE_RYEEZ6WJ
 
+#include <stdint.h>
+#include "kaHIP_config.h"
```

Comment:
Minor, but perhaps this:
```
#include "kaHIP_config.h"

#ifdef __cplusplus
#include <cstdint>
extern "C"
{
#else
#include <stdint.h>
#endif
```

I'm always wary about including the `C` headers instead of `C++` ones...